src/app/card/card.component.html

```diff
@@ -0,0 +1,24 @@
+<kendo-card width="320px" class="card-main">
+  <kendo-card-header class="k-hstack">
+    <div>
+      <h1 kendoCardTitle>{{ title ? title : "No Title" }}</h1>
+      <p kendoCardSubtitle>{{ name ? name : "Anonymous" }}</p>
+    </div>
+  </kendo-card-header>
+
+  <kendo-card-body>
+    <p>
+      {{ description ? sliceText(description) : "No Description" }}
```

Comment:
not good practice to call a function inside the template. Check the Rashmi's code and get an idea and refactor this.

Reply:
done